test/e2e/suite/carotation/carotation.go

```diff
@@ -135,30 +138,40 @@ var _ = framework.CasesDescribe("CA rotation", func() {
 		caData, ok := caSecret.Data["ca.crt"]
 		Expect(ok).To(BeTrue(), "expected 'ca.crt' to be present in Issuer CA Secret")
 
+		tlsData, ok := caSecret.Data["tls.crt"]
+		Expect(ok).To(BeTrue(), "expected 'tls.crt' to be present in Issuer CA Secret")
+
+		Expect(caData).To(Equal(tlsData), "invalid test; expected 'ca.crt' to equal 'tls.crt' for Issuer CA Secret (this implies a previous test run wasn't cleaned up correctly)")
```

Comment:
note: the changes in this file relate to the fact that this test modifies _global_ state (i.e. the secret containing the install-time CA). I tried to undo changes to global state in the test and to do a sanity check.